dependencies.gradle

```diff
@@ -81,7 +81,7 @@ libraries.mockitoJunit5 = "org.mockito:mockito-junit-jupiter"
 libraries.passay = "org.passay:passay:1.6.5"
 libraries.postgresql = "org.postgresql:postgresql:42.7.4"
 libraries.selenium = "org.seleniumhq.selenium:selenium-java:${versions.seleniumVersion}"
-libraries.seleniumHttp = "org.seleniumhq.selenium:selenium-http-jdk-client:4.13.0"
+libraries.seleniumRemoteDriver = "org.seleniumhq.selenium:selenium-remote-driver:${versions.seleniumVersion}"
```

Comment:
what is the rationale behind changing from Selenium HTTP client to Selenium Remote Driver?

Reply:
After issues with chrome driver some days ago I  analyzed again selenium and if there are solutions for:

we receive updates from dependabot but stuck with http version see https://mvnrepository.com/artifact/org.seleniumhq.selenium/selenium-http-jdk-client

this client seems to be outdated and with this PR I found the solution to allow again selenium updates. The outdated client was only working up to version 4.18